src/main/scala/com/fulcrumgenomics/identifyprimers/PrimerMatch.scala

```diff
@@ -111,7 +111,7 @@ private[identifyprimers] case class UngappedAlignmentPrimerMatch(primer: Primer,
 private[identifyprimers] case class GappedAlignmentPrimerMatch
 (primer: Primer, score: Double, secondBestScore: Double, start: Int, end: Int) extends PrimerMatch {
   require(score >= secondBestScore, s"second best score ($secondBestScore) must be less than or equal to the score ($score)")
-  require(start <= end)
+  require(start <= end, s"the alignment start($start) is bigger than the end($end) for ($primer)")
```

Comment:
```suggestion
  require(start <= end, s"The alignment start ($start) is bigger than the end ($end) for ($primer)")
```
capitalize the beginning of a sentence, and missing spaces before parens